openedx/core/djangoapps/content/search/tests/test_api.py

```diff
@@ -975,3 +1061,51 @@ def test_block_in_units(self, mock_meilisearch):
             ],
             any_order=True,
         )
+
+    @override_settings(MEILISEARCH_ENABLED=True)
+    def test_units_in_subsection(self, mock_meilisearch):
```

Comment:
```suggestion
    def test_units_in_subsection(self, mock_meilisearch) -> None:
```
Nit: it's best to put `-> None` on all your test case functions (it's missing on a few others), otherwise `mypy` won't type check them. Mypy only checks functions that have at least some type information somewhere in the definition.

Reply:
Thanks! That's interesting. I have updated it: https://github.com/openedx/edx-platform/pull/36762/commits/cbd5c84b826f3aeab9d902c0a8bd600d85ddd7d4